src/components/FormValidator.js

```diff
@@ -78,7 +78,7 @@ export default class FormValidator {
     this._submitButton = this._formElement.querySelector(
       this._submitButtonSelector
     );
-    console.log(this._formElement instanceof HTMLFormElement);
+    this._formElement instanceof HTMLFormElement;
 
     this._formElement.addEventListener("submit", (e) => {
       e.preventDefault();
```

Comment:
```js

      this._formElement.reset();
      this._inputEls.forEach((inputEl) => this._hideInputError(inputEl));
      this._disableSubmitButton();
```

Please, delete these 3 lines of code from `FormValidator.js`

* You need to **reset** the form only in a `then` block in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.
* You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.